package.json

```diff
@@ -2382,6 +2382,15 @@
                     "light": "images/light/icon-checkout.svg"
                 }
             },
+            {
+                "command": "gitlens.views.pruneRemote",
+                "title": "Prune Remote",
+                "category": "GitLens",
+                "icon": {
```

Comment:
I would suggest removing the icon altogether, since I don't think this command warrants an inline toolbar button. (Also branch doesn't feel quite right either).